include/envoy/buffer/buffer.h

```diff
@@ -189,6 +189,14 @@ class Instance {
    */
   virtual ssize_t search(const void* data, uint64_t size, size_t start) const PURE;
 
+  /**
+   * Search for an occurrence of a buffer at the start of a larger buffer.
```

Comment:
Search for an occurrence of a buffer at the start of a larger buffer. -> 
Search for an occurrence of data at the start of a buffer.

I think the comment above from could be updated too, given it's not searching for a buffer in a larger buffer.